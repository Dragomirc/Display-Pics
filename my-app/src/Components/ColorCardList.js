import React from "react";
import ColorCard from "./ColorCard";

const colors = ["red", "yellow", "green", "purple"];
const ColorCardList = props => {
  const colorItems = colors.map(color => {
    return <ColorCard color={color} info={props.info} />;
  });

  return <ul>{colorItems}</ul>;
};

export default ColorCardList;
