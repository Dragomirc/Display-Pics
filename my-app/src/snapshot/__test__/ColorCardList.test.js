import React from "react";
import ColorCardList from "../../Components/ColorCard";
import renderer from "react-test-renderer";

it("renders ColorCards", () => {
  const component = renderer.create(<ColorCardList info={() => 3} />);
  expect(component).toMatchSnapshot();
});
