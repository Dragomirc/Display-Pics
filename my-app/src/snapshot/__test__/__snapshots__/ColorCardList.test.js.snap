// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ColorCards 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
/>
`;
